src/status_im2/contexts/quo_preview/profile/collectible.cljs

```diff
@@ -0,0 +1,55 @@
+(ns status-im2.contexts.quo-preview.profile.collectible
+  (:require
+   [quo2.components.profile.collectible.view :as quo]
+   [react-native.core :as rn]
+   [reagent.core :as reagent]
+   [status-im2.contexts.quo-preview.preview :as preview]))
+
+(def descriptor
+  [{:label   "Number of images"
+    :key     :num-imgs
+    :type    :select
+    :options (map (fn [n]
+                    {:key   n
+                     :value (str n " images")})
+                  (range 1 11))}
+   {:label "Random order of images"
+    :key   :shuffle-imgs
+    :type  :boolean}])
+
+(defn cool-preview
+  []
+  (let [state       (reagent/atom {:num-imgs 1 :shuffle-imgs false})
+        test-images ["https://i.seadn.io/gcs/files/ddefc5cbae2a8bc5d5aa368ca5887a23.jpg"
+                     "https://i.seadn.io/gcs/files/4b639beda8da0c85b3fb0ce9ea1be1dd.png"
+                     "https://i.seadn.io/gcs/files/f8b544080d7af06f81537f35559d7f75.png"
+                     "https://i.seadn.io/gcs/files/52347f7cee9af5b6f217f7a1782b7295.webp"
+                     "https://i.seadn.io/gcs/files/5f9d314ebd514d3f4a02ce7836ed9f9c.webp"
+                     "https://i.seadn.io/gcs/files/cd2323f7c4548519ec5679f9bc8825e1.png"
+                     "https://i.seadn.io/gcs/files/d3781f86c3ff626070559d01a85b1f0f.png"
+                     "https://i.seadn.io/gcs/files/5c332aca9e14c4f7cbc35f48becc7441.png"
+                     "https://i.seadn.io/gcs/files/eeb644ff4384804f0e2158435ec8bd76.png"
+                     "https://i.seadn.io/gcs/files/f90220e9846e8be8ce9acfcc3db29f96.png"]]
+    (fn []
+      (let [imgs-to-show (cond->> test-images
+                                  (:shuffle-imgs @state)
+                                  (shuffle)
+                                  :always
+                                  (take (:num-imgs @state)))]
+        [rn/view
+         {:margin-bottom 50
+          :padding       16}
+         [preview/customizer state descriptor]
+         [rn/view
+          {:padding-vertical 100
+           :align-items      :center}
+          [quo/collectible {:imgs imgs-to-show}]]]))))
+
+(defn preview-collectible
+  []
+  [rn/view {:style {:flex 1}}
+   [rn/flat-list
+    {:flex                      1
+     :keyboardShouldPersistTaps :always
```

Comment:
I believe there's no need to use camel case for top level keys passed to components transformed via `reagent/adapt-react-class` (which is the case for `rn/flat-list`).

Reply:
Yep, you're right.

I'm not sure how exactly is the preview module implemented, so I was copying the structure already used in other preview screens, and this is something I didn't notice. The change is in 2e25e76cc0a31eebad5d183bbb8963d3a8a02681

Reply:
Yeah @ulisesmac, I see other places unnecessarily using camel case, but good thing you changed in your PR, thanks. In fact, as you probably noticed, all preview screens are very similar and there's a lot of copy and paste going on to set up each one. Eventually we should be able to abstract the preview screens a bit better.